File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/non-plagiarized/14/Soal6.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function reverse(number: string): void {
  for (let i = number.length - 1; i >= 0; i--) {
    console.log(number.charAt(i));
  }
}

rl.question('Read a number: ', (input) => {
  let numbers = '';
  for (let i = 0; i < 10; i++) {
    rl.question('Read a number: ', (num) => {
      numbers += num;
      if (i === 9) {
        reverse(numbers);
        rl.close();
      }
    });
  }
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/non-plagiarized/14/Soal6.py
import sys

def reverse(number):
    for i in range(len(number)-1, -1, -1):
        print(number[i])

if __name__ == "__main__":
    input_str = ""
    for i in range(10):
        a = int(input("Read a number: "))
        input_str += str(a)
    reverse(input_str)

